src/util/condition_tools.py

```diff
@@ -124,6 +124,22 @@ def created_announcements_for_conditions_dict(
     return output_announcements
 
 
+def created_announcement_names_for_conditions_dict(
```

Comment:
Why reimplement the method instead of using the same method?
[an.name() for an in created_announcements_for_conditions_dict(d, name)]

Reply:
Done